### ISSUE TITLE
Include script to initialize new projects  

### ISSUE THREAD

**Issue by laurianvm** [vliz-be-opsci/k-gap#1]
It would be useful to be able to initialize new k-gap project by running some script that clones this repository & asks some questions to set up the project context (structure / naming of folders (for configs, templates, input-data, ...) and makes correct connections in the docker environement) 
similarly to how npm init runs; 
e.g. by adding '| bash' to a command (as used in [marc's dotfiles](https://github.com/marc-portier/dotfiles))

**Comment by marc-portier** [comment]
agree, the goal should be to document high up in the README.md the usage of

`curl -sL https://raw.githubusercontent.com/vliz-be-opsci/kgap/main/kgap-init | bash`

in fact, additionally, provided we have the proper linking under the website publication setup correctly that should become:

`curl -sL https://open-science.vliz.be/kgap/kgap-init | bash`

**Comment by marc-portier** [comment]
additionally it could create boilerplate folders and some useful typical content:

./data
./config
./notebooks/requirements.txt
.gitignore
.env
docker-compose.yml

**Comment by marc-portier** [comment]
also: think it is smart to have this script suggest using a project-specific REPO_NAME inside graphdb -- so that not everyone is using the kgap default there

**Comment by laurianvm** [comment]
relating to the boilerplate folders & their useful typical content --> listing all 'allowed' content would be handy for practical usage and could serve as 'how to'-documentation within kgap projects
